filterprotobuffer.go

```diff
@@ -0,0 +1,55 @@
+package dapper
+
+import (
+	"bytes"
+	"github.com/dogmatiq/iago/indent"
+	"github.com/dogmatiq/iago/must"
+	"google.golang.org/protobuf/encoding/prototext"
+	"google.golang.org/protobuf/proto"
+	"io"
+	"reflect"
+)
+
+// protobufMessageType is the reflect.Type for proto.Message
+var protobufMessageType = reflect.TypeOf((*proto.Message)(nil)).Elem()
+
+// ProtobufferFilter is a filter for proto.Message types. It shows the field names as defined in the .proto file
+// and hides implementation-specific internal state.
+func ProtobufferFilter(
+	w io.Writer,
+	v Value,
+	c Config,
+	p FilterPrinter,
+) (err error) {
```

Comment:
```suggestion
) error {
```

We prefer to [avoid named parameters](https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values), unless doing one of two things:

- modifying the value in a `defer`
- referencing them by name in documentation (usually when there are multiple return values)